lib/call.js

```diff
@@ -112,6 +112,31 @@ var Call = function (client) {
 		return this.update(callId, { state : "active" }).asCallback(callback);
 	};
 
+	/**
+	 * Transfer a call
+	 * @param {Object} params Parameters for transfering of the call
+	 * @param {String} params.transferTo Phone number or SIP address that the call
+	 * is going to be transferred to.
+	 * @param {String} params.transferCallerId The caller id that will be used when the call is transferred
+	 * @param {Object} params.whisperAudio Audio to be played to the caller that the call will be transferred to.
+	 * @param {Function} [callback] Callback with the transfered call
+	 * @return {CallResponse} A promise for the transfered call
+	 */
+	this.transfer = function (callId, params, callback) {
+		params.state = "transferring";
+		return client.makeRequest({
+			path   : "calls/" + callId,
+			method : "POST",
+			body   : params
+		})
+		.then(function (response) {
```

Comment:
Can you pull this out into it's own function somewhere. I think we will probably use this for other methods. Fairly useful :)